packages/state/src/beacon/WalletClient.ts

```diff
@@ -10,3 +11,12 @@ export const WalletClient =
         iconUrl: "",
         appUrl: "https://umamiwallet.com/",
       });
+
+export const resetWalletConnection = (persistor: Persistor) =>
```

Comment:
I think a better name would be like logout because it not only reset the wallet connection, but also logs you out